### PR TITLE
Update RaspberryPiInstructions.md

### PR DESCRIPTION
**Pull request by hqueue**
Add more detailed step.

### CODE REVIEW

File: samples/RaspberryPiInstructions.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ These steps have been tested on a RPi 2 and RPi 3 with Linux and Windows.
 
 * From the terminal/commandline create a folder named helloworld and go into it.
 * Run `dotnet new console -n helloworld`.
+* Change directory into `helloworld` and you can find helloworld.csproj file.
```
Comment (Petermarcu) [inline samples/RaspberryPiInstructions.md]: If we're going to tell people here to change to the helloworld directory, we should delete the words telling people to do that from two lines higher.

**Comment by Petermarcu** [comment]
One piece of feedback, then I think this is good to merge. Thanks!

File: samples/RaspberryPiInstructions.md (hunk 1 above)
Comment (hqueue) [inline samples/RaspberryPiInstructions.md]: You are right, I will update it. 
With recent SDK, if I ran `dotnet new console -n helloworld`, a new directory `helloworld` is created under current directory. I will update this paragraph.

**Comment by hqueue** [comment]
@Petermarcu PTAL. I've removed `-n helloworld` option from dotnet command, since this option make a new  `helloworld` directory under current directory. And also refine the paragraph.

**Pull request merged**